Gemfile.lock

```diff
@@ -259,11 +259,12 @@ GEM
     zeitwerk (2.4.2)
 
 PLATFORMS
-  x86_64-darwin-20
+  x86_64-linux-musl
 
 DEPENDENCIES
   github-pages (~> 214)
+  jekyll-github-metadata
   webrick (~> 1.7)
 
 BUNDLED WITH
-   2.2.17
+   2.2.2
```

Comment:
isn't 2.2.2 lower than 2.2.17 ?

Reply:
it is, but this is the version of bundler in `jekyll/jekyll` image (https://hub.docker.com/r/jekyll/jekyll)